rancher/distros-test-framework
# Adding test for flannel version bump

#### Proposed Changes ####


Added separate test for flannel cni



#### Types of Changes ####


Added new test for flannel cni

#### Testing ####


Checklist:
1. If your PR changes anything on or related to Jenkins, run it pointing to your branch to make sure it's okay.
No

2. Verify code lint; we should not have errors.


3. Update the documentation if needed.


4. Update makefile and docker run if adding new tests.


5. Run your tests at least 4 times with all configurations needed and possible.


6. If needed test with different os types.


#### Linked Issues ####



#### Further Comments ####
